README.md

```diff
@@ -151,3 +151,14 @@ helm […] --devel trustify/<chart> […]
 ## Initial set of importers
 
 You can create an initial set of importers by adding the values file `values-importers.yaml`.
+
+## External resources embedded
+
+We use a local copy of the following resources to be able to use in disconnected environments
+
+```bash
```

Comment:
If you make those actual `curl` commands, the user would only need to run this. Or click on the run icon in IntelliJ (probably vscode too).

Reply:
Done with wget